docs/group_faqs/gf_faqs.rst

```diff
@@ -1,21 +1,19 @@
 Frequently Answered Questions
 =============================
 
-What are plans in Solace PubSub+ cloud?
+What are service types in Solace PubSub+ cloud?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-On PubSub+ Cloud you could choose a plan to use for your messaging needs, below are the enterprise plans PubSub+ cloud offers:
+On PubSub+ Cloud you could choose a service type to use for your messaging needs, below are the enterprise service types PubSub+ cloud offers:
```

Comment:
Say "below are the service types available in Enterprise accounts."

Reply:
Actually it would be good if you could have two sections under here, one for Standard Accounts and one for Enterprise Accounts and list the service types in each without talking about service classes.